go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	github.com/mitchellh/hashstructure/v2 v2.0.2
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/np-guard/cluster-topology-analyzer v1.7.0
-	github.com/np-guard/netpol-analyzer v0.2.2
+	github.com/np-guard/netpol-analyzer v0.4.2
```

Comment:
How about splitting this PR into update and new command?

Reply:
closing this PR, will split to two separate PRs.